tidb-configuration-file.md

```diff
@@ -212,12 +219,13 @@ Configuration items related to log.
 - Outputs the threshold value of consumed time in the slow log.
 - Default value: `300ms`
 - If the value in a query is larger than the default value, it is a slow query and is output to the slow log.
+- Since v6.1.0, the threshold value of consumed time in the slow log is determined by configuration [`instance.tidb_slow_log_threshold`](/tidb-configuration-file.md#tidb_slow_log_threshold) or system variable [`tidb_slow_log_threshold`](/system-variables.md#tidb_slow_log_threshold). `slow-threshold` still takes effect. But if `slow-threshold` and `instance.tidb_slow_log_threshold` are setting at the same time, TiDB will apply the value of `instance.tidb_slow_log_threshold`.
 
 ### `record-plan-in-slow-log`
 
-+ Determines whether to record execution plans in the slow log.
-+ Default value: `1`
-+ `0` means to disable, and `1` (by default) means to enable. The value of this parameter is the initial value of the [`tidb_record_plan_in_slow_log`](/system-variables.md#tidb_record_plan_in_slow_log) system variable.
+- Determines whether to record execution plans in the slow log.
+- Default value: `1`
+- Since v6.1.0, the threshold value of consumed time in the slow log is determined by configuration [`instance.tidb_record_plan_in_slow_log`](/tidb-configuration-file.md#tidb_record_plan_in_slow_log) or system variable [`tidb_record_plan_in_slow_log`](/system-variables.md#tidb_record_plan_in_slow_log). `record-plan-in-slow-log` still takes effect. But if `record-plan-in-slow-log` and `instance.tidb_record_plan_in_slow_log` are setting at the same time, TiDB will apply the value of `instance.tidb_record_plan_in_slow_log`.
```

Comment:
```suggestion
- Since v6.1.0, whether to record execution plans in the slow log is determined by the TiDB configuration item [`instance.tidb_record_plan_in_slow_log`](/tidb-configuration-file.md#tidb_record_plan_in_slow_log) or the system variable [`tidb_record_plan_in_slow_log`](/system-variables.md#tidb_record_plan_in_slow_log). `record-plan-in-slow-log` still takes effect. But if `record-plan-in-slow-log` and `instance.tidb_record_plan_in_slow_log` are set at the same time, TiDB will apply the value of `instance.tidb_record_plan_in_slow_log`.
```